framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeMirror.java

```diff
@@ -657,12 +649,11 @@ public boolean removeAnnotation(AnnotationMirror a) {
      * @return true if the annotation was removed, false if the type's annotations were unchanged
      */
     public boolean removeAnnotation(Class<? extends Annotation> a) {
```

Comment:
Consider renaming this to `removeAnnotationByClass`, analogous to `getAnnotationByClass`, to avoid confusion with `removeAnnotation`.